src/std/net/socket/kqueue-server.ss

```diff
@@ -102,15 +102,15 @@ package: std/net/socket
                 (if io-in
                   (kqueue-kevent-add kq sock EVFILT_READ)
                   (begin
-                    (kqueue-kevent-del kq sock)
+                    #;(kqueue-kevent-del kq sock)
                     (hash-remove! fdtab (fd-e sock))))
                 (set! (!socket-wait-out ssock) #f)
                 (set! (!socket-state-io-out state) #f)
                 (close-io-out! io-out sock)
                 (unless io-in
                   (close-port sock)))
                ((inout)
-                (kqueue-kevent-del kq sock)
+                #;(kqueue-kevent-del kq sock)
```

Comment:
I guess this really can be optimized away because the fd is closing?
It's nice that kqueue doesn't error with `EBADF` and automatically removes the file descriptor from the event set.